Sources/TuistAutomation/XcodeBuild/XcodeBuildController.swift

```diff
@@ -181,7 +181,16 @@ public final class XcodeBuildController: XcodeBuildControlling {
         output: AbsolutePath
     ) -> AsyncThrowingStream<SystemEvent<XcodeBuildOutput>, Error> {
         var command = ["/usr/bin/xcrun", "xcodebuild", "-create-xcframework"]
-        command.append(contentsOf: frameworks.flatMap { ["-framework", $0.pathString] })
+        command.append(contentsOf: frameworks.flatMap {
+            let pathString = $0.pathString
+            return [
+                "-framework",
+                // It's workaround for Xcode 15 RC bug
+                // remove it since bug will be fixed
+                // more details here: https://github.com/tuist/tuist/issues/5354
```

Comment:
Is this an Xcode bug? Do you have any reference?

Reply:
No I do not have any references

Reply:
But you could ensure this bug happens to try any input framework from var folder (temp folder)

Reply:
I can confirm is a bug in Xcode 15 when the paths of the frameworks being combined into an XCFramework are symlinks. In this case are because `/var/folders` is a symlink to `/private/var/folders`. I reproduced the issue [here](https://github.com/tuist/xcode-15-xcframework-bug) and filed a [radar](https://feedbackassistant.apple.com/feedback/13191683)